### PR TITLE
Phoenix deployment example issue#17

### PR DESCRIPTION
**Pull request by nelsonic**
I got tired of "piecing" together Phoenix Heroku Deployment from our various projects, 
so I decided to create a _step-by-step_ guide.
(_I needed to **reference** it in: https://github.com/dwyl/phoenix-chat-example#deployment ..._)

Please review/merge when happy. thanks.

Works:
phoenix-chat-on-heroku

### CODE REVIEW

**Comment by nelsonic** [comment]
phoenix-chat-on-heroku

**Comment by nelsonic** [comment]
phxchat

File: elixir-phoenix-app-deployment.md [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+
+Also in the root directory of your App,
+create a file called `Procfile`
+
+_Paste_ this line in the file:
+
+```sh
+web: MIX_ENV=prod mix ecto.migrate && mix phx.server
+```
+That will ensure that your database tables/schema is up-to-date
```
Comment (iteles) [inline elixir-phoenix-app-deployment.md:133]: Big fan of these types of lines. Not just 'copy paste this' but 'this is what this does'.

**Review by iteles — APPROVED** [review]
I hadn't realised it was this 'involved' - hard to  understand why this is the first time we've written it up! 👍

**Comment by iteles** [comment]
Leaving `in-review` label as I know @Cleop is currently reviewing .

**Comment by Cleop** [comment]
I wanted to change this typo on the main readme file:
image
But I don't have push access for this repo.... @nelsonic - can you grant me access or make the change?

File: elixir-phoenix-app-deployment.md [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+![heroku-postgres-select-default](https://user-images.githubusercontent.com/194400/36438710-9ae7b2ea-1662-11e8-829b-c1ee854b54d5.png)
+
+Leave the _default_ "Hobby Dev - Free"
+and _click_ the "**Provision**" button.
+
+You should now see (_something similar to_) the following: <br />
+![heroku-addon-installed](https://user-images.githubusercontent.com/194400/36438893-21945f00-1663-11e8-87a4-62baa1f56e95.png)
+
+
+### 5. Create a `elixir_buildpack.config` File
```
Comment (SimonLab) [inline elixir-phoenix-app-deployment.md:110]: I think I would put this section just before (or after) the section 9 where the user add the buildpacks manually to Heroku.
I would also add more explanation on what are buildpacks

**Review by SimonLab — APPROVED** [review]
Looks good :+1: 
One comment on the buildpacks sections

**Comment by nelsonic** [comment]
@Cleop / @SimonLab you have Write access so you can make any changes. 
If you _prefer_ simply create an Issue and either come back to it or let someone `else` resolve.

**Review by Cleop — APPROVED** [review]
Great idea to record all this. I've changed the typo so looks ready to go 🏁

**Comment by nelsonic** [comment]
@Cleop & @SimonLab _thank you_. ❤️ ✅

**Pull request merged**